Localize/lang/strings.json

```
@@ -935,6 +938,7 @@
   "_4wjJs0.comment": "Hour of the day",
   "_5+MuPz.comment": "Error validation message",
   "_5FINOY.comment": "Error message to show when there are invalid connections in the nodes.",
+  "_5HGtu7.comment": "Message on deleting connection that exists in multiple spots",
```

Not quite sure what this means?

```suggestion
  "_5HGtu7.comment": "Message to show when deleting a connection that exists in multiple places.",
```